app/javascript/dashboard/components/widgets/conversation/conversationCardComponents/CardLabels.vue

```diff
@@ -2,10 +2,11 @@
   <div
     v-show="activeLabels.length"
     ref="labelContainer"
-    class="label-container mt-0.5 mx-2 mb-0"
+    v-resize="computeVisibleLabelPosition"
+    class="w-4/5 label-container"
```

Comment:
Let's not add a `w-4/5` here, we can add this from the parent, where this component is used. Just as a rule of thumb, a component should not dictate it's rendering in the layout, and should only take care of the layout within it's children. It makes the component reusable.